privacyidea/lib/tokens/smstoken.py

```diff
@@ -320,7 +320,7 @@ def create_challenge(self, transactionid=None, options=None):
                                                         "{0!s}_{1!s}".format(self.get_class_type(),
                                                                              ACTION.CHALLENGETEXT),
                                                         options) or _("Enter the OTP from the SMS:")
-        attributes = {'state': transactionid}
+        attributes = {'attributes': {'state': transactionid}}
```

Comment:
Please rename the variable as ``reply_dict``

Reply:
Done in db7cb708